plugins/exec.lua

```diff
@@ -11,6 +11,19 @@ local function exec(cmd, keep_newline)
 end
 
 
+local function exec_rw(dv, cmd, keep_newline)
+  local proc = process.start { "sh", "-c", cmd }
+  proc:write(tostring(dv))
```

Comment:
Does `tostring(dv)` do what you expect?

Reply:
It works as I expect: It transforms the content to a string (no extra newlines, nothing strange...)
Do you suggest a different approach?

Reply:
Finally, as you suspected, it didn't work. I've removed `dv` references and worked in a more direct approach.